pinot-core/src/test/java/org/apache/pinot/core/data/manager/TableIndexingTest.java

```diff
@@ -297,6 +297,9 @@ public void testAddIndex(TestCase testCase) {
               ...
             } */
           // no params
+          if (field.getDataType() == DataType.BOOLEAN) {
+            throw new IllegalArgumentException("Bloom filter index is not supported for BOOLEAN type");
+          }
```

Comment:
Should we add this check into the test? Shouldn't it be thrown from the production code to be verified?